src/main/java/htsjdk/samtools/SamReaderFactory.java

```diff
@@ -1,528 +1,528 @@
-package htsjdk.samtools;
```

Comment:
The file permissions here got changed making it look like a huge diff in this file.  Lets put it back to how it was.